i18n/ku.toml

```diff
@@ -0,0 +1,11 @@
+[writtenBy]
+other = "Nivîskar:"
+
+[on]
+other = "li"
+
+[generator]
+other = "Bi <a href='https://gohugo.io'>Hugo</a> using the <a href='https://github.com/EmielH/tale-hugo/'>Tale</a>-ê hatiye çêkirin."
```

Comment:
Hi @rojserbest, thanks a lot for these translations! It looks like you missed "using the" in this line. 🙂

Reply:
Sorry, my bad. I fixed it :)